src/pages/Login.vue

```diff
@@ -53,6 +50,22 @@ export default {
       return to
     }
   },
+  mounted () {
+    // eslint-disable-next-line no-undef
+    google.accounts.id.initialize({
+      client_id: '518573655847-qbtbogn8jr77m3p5q7rot2m29vrurkhb.apps.googleusercontent.com',
```

Comment:
where we get client id?? If this is a google id for our project should be on .env

Reply:
I'm goint to change that to use a .env variable